spec/unit/resource/attributes_spec.rb

```diff
@@ -32,6 +32,23 @@ module ActiveAdmin
         expect(subject).to_not include :published_date
         ActiveAdmin.application.filter_attributes = keep
       end
+
+      context "counter cache" do
+        subject do
+        end
+
+        it "should not include counter cache columns" do
+          attrs = ActiveAdmin::Resource.new(namespace, Category).resource_attributes
+
+          expect(attrs.keys).not_to include(:posts_count)
+        end
+
+        it "should include counter cache like columns" do
+          attrs = ActiveAdmin::Resource.new(namespace, User).resource_attributes
+
+          expect(attrs).to include(sign_in_count: :sign_in_count)
+        end
+      end
```

Comment:
This is nitpicky so I'll leave it up to you how best to address. Since it's already using a subject, it could help to have a new context since one of the tests is for a `"column with 'counter-cache'-like name"` context vs a `"counter cache column"` context.

Reply:
Thanks! Good catch. At the beginning I tried to use just User model but it got hard. I forgot to remove the subject but now I think I'm gonna use it as you suggested (two contexts)